package.json

```diff
@@ -4,8 +4,9 @@
   "description": "Alternative syntax for WebAssembly text format",
   "main": "index.js",
   "scripts": {
+    "test": "npm run lint && npm run flow && npm run ava",
```

Comment:
I'm not a fan of having the `test` script also run code validation. I added the separate `validation` npm script just for this reason. Let's use that instead.

Reply:
I've seen this in every other project I looked at in the past, that's why I added it

Reply:
Yeah, I can see why you added it. I think that's odd that they do that TBH. If I was exploring a new repo I'd expect `test` to run unit tests not the linter ;p. I'd rather they be two discreet scripts. Travis has a `before_script` which would be perfect for `npm run validate`.

Reply:
I think it's because that way you have a single script to execute, mostly beneficial outside of ci and even there it prevents you from forgetting to execute it